<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatShip</class>
 <widget class="QDialog" name="CreatShip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建矢量文件</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 255, 206);</string>
  </property>
  <widget class="QLineEdit" name="RasterName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;宋体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ShapName">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;宋体&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenRasterBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 246);
font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>打开栅格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveShapeBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 246);
font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>保存矢量</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRows">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>划分行数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCols">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>划分列数：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 246);
font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 246);
font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>划分行列数</string>
   </property>
   <widget class="QSpinBox" name="SpinBoxRows">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>31</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="RowsFilter" stdset="0">
     <UInt>0</UInt>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinBoxCols">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="FilterCols" stdset="0">
     <UInt>0</UInt>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>SaveShapeBtn</zorder>
  <zorder>ShapName</zorder>
  <zorder>RasterName</zorder>
  <zorder>OpenRasterBtn</zorder>
  <zorder>labelRows</zorder>
  <zorder>labelCols</zorder>
  <zorder>CreateBtn</zorder>
  <zorder>CancelBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
